types/index.d.ts

```diff
@@ -36,7 +38,7 @@ export declare class Store<S> {
   }): void;
 }
 
-export declare function install(Vue: typeof _Vue): void;
+export function createStore<S>(options: StoreOptions<S>): Store<S>;
```

Comment:
Look forward to this!

Currently in the `alpha.1`, I'm facing this error on using `vuex` with typescript:

```
ERROR in /home/chenxeed/Projects/chenxeed/chenxeed-v2/src/store/index.ts(2,10):
2:10 Module '"../../node_modules/vuex/types"' has no exported member 'createStore'. Did you mean to use 'import createStore from "../../node_modules/vuex/types"' instead?
  > 1 | import { createStore } from "vuex";
```

I believe this should fix this error.

Reply:
Yes, typings are not yet supported at `alpha.1`. This should do it 👍